tests/InstrEngineTests/NaglerInstrumentationMethod/stdafx.h

```diff
@@ -46,4 +46,10 @@ using namespace ATL;
     do { if (FAILED(hr = (EXPR))) { ATLASSERT(FALSE); return hr; } } while (false)
 #endif
 
-#include "../../../src/Common.Headers/tstring.h"
\ No newline at end of file
+#include "../../../src/Common.Headers/tstring.h"
+#include "../../../src/Common.Lib/Macros.h"
+#include "../../../src/Common.Lib/refcount.h"
+#include "../../../src/InstrumentationEngine.Lib/ImplQueryInterface.h"
```

Comment:
We shouldn't be including anything from InstrumentationEngine.Lib since we will not link to that library. Is there a way that we can share this code instead? Maybe just move the header into the common headers?